source/Running.js

```diff
@@ -130,6 +130,52 @@ export default class Running extends Component {
       borderRadius: "0.3rem",
     };
 
+    const prolificCreateFraftOnClick = async () => {
+      const prolificStudyDraftApiUrl =
+        "https://api.prolific.co/api/v1/studies/";
+      const payload = {
+        name: user.currentExperiment.titleOfStudy,
+        internal_name: this._getPavloviaExperimentUrl(),
+        description: user.currentExperiment.descriptionOfStudy,
+        external_study_url: user.currentExperiment.experimentUrl,
+        prolific_id_option: "url_parameters",
+        completion_code: completionCode,
+        completion_option: "url",
+        total_available_places: user.currentExperiment._participantsHowMany,
+        estimated_completion_time:
+          user.currentExperiment._participantDurationMinutes,
+        reward: 1,
+        device_compatibility: [],
+        peripheral_requirements: [],
+        eligibility_requirements: [],
+      };
+
+      const response = await axios.post(
+        prolificStudyDraftApiUrl,
+        JSON.stringify(payload),
+        {
+          withCredentials: false,
+          headers: {
+            "Content-Type": "application/json",
+            "Access-Control-Allow-Origin": "*",
+            Authorization:
+              "Token XlJQZJfE1vbw3nDXvrL64p1cFuvEgnDTr1eo1WzhiZ-XCNCt9wsqXomi7nhUba6UVsvRXedtvUcTzFTAyaScwQvY5e_nvgxfX_6QnXHeECchPcWOULjg9rGJ",
```

Comment:
Currently I have added my own token to validate requests. But it would be better if we could use someone elses in the environment variables and use it here. Is it already set somewhere in the env files, couldn't find any

Reply:
Just saw this comment through email. I think you should never expose your token in the code on GitHub? Can you remove it and revoke the access through Prolific?

If I understand it correctly - we would need a token to access and use Prolific API. Is it related to an APP account, or a researcher account? Do they have a quota limit? We can schedule a meeting to discuss how to enable this later, but I strongly suggest you revoke the whole commit or revoke the token completely through Prolific now. @ritikamotwani

Reply:
yes, you're right. I added a comment to bring this to notice, but forgot this is not a private repo. Let me remove this part in a bit.